barretenberg/cpp/CMakePresets.json

```diff
@@ -375,6 +375,7 @@
         "WASI_SDK_PREFIX": "/opt/wasi-sdk",
         "CC": "$env{WASI_SDK_PREFIX}/bin/clang",
         "CXX": "$env{WASI_SDK_PREFIX}/bin/clang++",
+        "CXXFLAGS": "-DBB_VERBOSE",
```

Comment:
I made verbose logging the default for convenience and so we have a record of the WASM benchmark in the CI (until there is a better way in the future, at least).